Sources/Sentry/SentrySpan.m

```diff
@@ -33,12 +34,23 @@ - (instancetype)initWithTracer:(SentryTracer *)tracer context:(SentrySpanContext
 
 - (id<SentrySpan>)startChildWithOperation:(NSString *)operation
 {
+    if (self.isFinished) {
+        SENTRY_LOG_WARN(
+            @"Starting a child on a finished span is not supported; it won't be sent to Sentry.");
+        return [SentryNoOpSpan shared];
+    }
```

Comment:
m: I think we dont need this logic here since we have it in `startChildWithOperation:description:`

Reply:
Ah of course, good point. Not sure why I missed that 😅